sdk/src/wallet/operations/transaction/high_level/send_native_tokens.rs

```diff
@@ -133,11 +131,9 @@ where
         let rent_structure = self.client().get_rent_structure().await?;
         let token_supply = self.client().get_token_supply().await?;
 
-        let account_addresses = self.addresses().await;
-        let default_return_address = account_addresses
-            .into_iter()
-            .next()
-            .ok_or(Error::FailedToGetRemainder)?;
+        let wallet_address = self.address().await;
+        // TODO: not needed anymore?
```

Comment:
Why not?

Reply:
just meant the variable binding, but I can also keep it.

Reply:
Ah yeah you can also remove the variable if you want, but at least remove the comment :D

Reply:
Done